Exercise17.md

```diff
@@ -0,0 +1,3 @@
+## Dự đoán kết quả, giải thích
+Thật sự dự đoán thì sai rồi, ko nghĩ nó sẽ lỗi.
```

Comment:
Thử mấy cái này xem được ý gì không 

1. Thử nếu không có dòng `Console.WriteLine(string.Join(" ", words));` thì còn lỗi không 
2. Thử giải thích cho chính xác hàm `Where` làm việc gì trong khối lệnh for (Làm việc gì chứ không phải là làm gì, cách thức chứ không phải mục đích)
3. Khi nào thì biểu thức so sánh `x != badwords[i]` được gọi
4. Thử tạo biến trung gian `var ch = badwords[i];` rồi sửa biểu thức so sánh trên xem còn lỗi không